server/src/main/java/fr/uga/l3miage/tp4/models/EcosSessionEntity.java

```diff
@@ -0,0 +1,26 @@
+package fr.uga.l3miage.tp4.models;
+
+
+
+import fr.uga.l3miage.tp4.enums.SessionStatus;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class EcosSessionEntity {
+    @Id
+    private Long id;
+    @Column(unique = true)
+    private String name;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
+
+    @Enumerated(EnumType.STRING)
+    private SessionStatus status;
+    @OneToOne
+    private EcosSessionProgrammationEntity programmeur;
```

Comment:
Attention à vos nom d'attribut
```suggestion
    @OneToOne
    private EcosSessionProgrammationEntity programmation;
```